src/suricata-common.h

```diff
@@ -40,6 +40,11 @@
 #include <autoconf.h>
 #endif
 
+#ifdef HAVE_DPDK
+#define HAVE_STRLCAT 1
+#define HAVE_STRLCPY 1
```

Comment:
Why is this here ?
Should not these `HAVE_STRLCPY` be set by configure in config.h ?

Reply:
Fixed, I moved STRLCAT/STRLCPY to configure.ac